### DISCUSSION TITLE
Cannot start Blender app in MacOS VM

### DISCUSSION THREAD

**Discussion by marcinwalus** [utmapp/UTM#5119] [UTM Help (macOS)]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
I created VM in UTM using "virtualise feature" and installed Blender app on it (I tried both arm and inter versions, version 3.4 and 2.9) and I cannot start it. App crashes during startup. 

**Configuration**  
* UTM Version: 4.2.1
* macOS Version: host and client Ventura 13.2.1 (22D68)
* Mac Chip (Intel, M1, ...): Apple M1 Max

**Crash log**  
Blender-2023-03-15-081649

**Debug log**  
N/A

**Upload VM**  
N/A
[Blender-2023-03-15-081649.ips.zip](https://github.com/utmapp/UTM/files/10977039/Blender-2023-03-15-081649.ips.zip)
![Screenshot 2023-03-15 at 08 28 45](https://user-images.githubusercontent.com/1926917/225237346-3a854713-6d2d-414f-86c6-f3b27991324b.png)

**Comment by osy** [comment]
I think this is an Apple/Blender issue. There isn’t much we can do here.
